bin/tessel-2.js

```diff
@@ -7,6 +7,7 @@ var key = require('../lib/key');
 var init = require('../lib/init');
 var logs = require('../lib/logs');
 
+
```

Comment:
I think you can write a rule for this in jscs.  so it will fail travis builds.. or are you only using jshint